MbDotNet/Models/Others/Config.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace MbDotNet.Models.Others{
```

Comment:
Instead of an `Others` namespace, I would just put this in the base `MbDotNet.Models` namespace rather than attempting to categorize it further.